katdal/test/test_sensordata.py

```diff
@@ -71,6 +71,20 @@ def test_numpy_object(self):
         np.testing.assert_array_equal(to_str(a), b)
 
 
+@mock.patch('katsdptelstate.encoding._allow_pickle', True)
```

Comment:
I haven't checked if it's necessary here, but in telstate unit tests that do unpickling I've done this to prevent the test emitting a warning:
```suggestion
@mock.patch('katsdptelstate.encoding._allow_pickle', True)
@mock.patch('katsdptelstate.encoding._warn_on_pickle', False)
```

Reply:
I haven't seen any warnings during testing but I'll add this anyway.